aspnetcore/signalr/security.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Security Considerations in ASP.NET Core SignalR
+author: rachelappel
+description: Learn how to use authentication and authorization in ASP.NET Core SignalR.
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 06/29/2018
+uid: signalr/security
+---
+
+# Security Concepts in ASP.NET Core SignalR
+
+By [Andrew Stanton-Nurse](https://twitter.com/anurse)
+
+## Overview
+
+SignalR provides a number of security protections by default, but it is important to understand their limitations.
```

Comment:
Split this into 2 sentences. I suggest the following:
"SignalR provides a number of security protections by default. It's important to understand their limitations."